lib/core.js

```diff
@@ -183,9 +186,17 @@ exports.dlTranspiler = function(transpilerName, update) {
       else
         installObj['core-js'] = tPackages['core-js'];
     }
-
+    
     // just do a quick install which checks basic existence
-    return install.install(installObj, { quick: !update, dev: true, summary: false });
+    var options = { quick: !update, dev: true, summary: false };
+    if (transpilerName === 'typescript') {
+      // specify shim configuration for typescript package
+      // having shim implies that 'typescript' module will have 'global' format 
+      options.override = {
+        shim: { 'bin/typescript': { 'exports': 'ts' } }
+      };
```

Comment:
This needs to be committed to the jspm registry rather as a shared override - https://github.com/jspm/registry/.